myapp/tests/example.py

```diff
@@ -0,0 +1,25 @@
+from django.test import TestCase, Client
```

Comment:
Note that in order to be "seen" by Django as test file. It needs to have the name "test_*.py" (I imagine you probably already know this...?)

Reply:
yes, due to it's an `example` test (for us to see how the content would look like), I removed the prefix to not running in the actual test; on the other side, as your suggestion, to keep the prefix in order to see how test files named
there is a tradeoff here :)